src/main/java/com/mixpanel/android/viewcrawler/ViewCrawler.java

```diff
@@ -713,9 +733,7 @@ private void updateEditState() {
             }
 
             {
-                final int size = mEditorChanges.size();
-                for (int i = 0; i < size; i++) {
-                    final Pair<String, JSONObject> changeInfo = mEditorChanges.get(i);
+                for (Pair<String, JSONObject> changeInfo:mEditorChanges.values()) {
```

Comment:
Unfortunately it's relatively inefficient (often it ends up allocating a bunch of new objects) so we prefer olde schoole C-style for loops when they're available.